src/bloks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-content: center;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px #f3f3f3 solid;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin-left: 3px;
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin-bottom: 4px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__review {
+  margin-bottom: 24px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 16px;
+  color: #060b35;
+}
+
+.card_priceTitle {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+  align-items: center;
+}
+
+.card_priceNumber {
+  margin-bottom: 16px;
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__link {
+  display: flex;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  width: 168px;
```

Comment:
better add horizontal paddings for cards and then you will not need to set fix width

Reply:
Ok.

Reply:
Everything has been fixed.